mega-linter-plugin-nitpick/nitpick.megalinter-descriptor.yml

```diff
@@ -0,0 +1,30 @@
+descriptor_id: NITPICK
+descriptor_type: tooling_format
+descriptor_flavors:
+  - all_flavors
+file_extensions:
+  - "*"
+active_only_if_file_found:
+  - ".nitpick.toml"
+  - "pyproject.toml"
+linters:
+  - linter_name: nitpick
+    linter_url: https://nitpick.readthedocs.io/
+    name: NITPICK
+    install:
+      dockerfile:
+        - RUN pip install nitpick==0.32.0
```

Comment:
Yes, there is a 0.32.0, *but there is no MegaLinter plug-in definition on that tag*.  The docs in this PR tell you to add `https://raw.githubusercontent.com/andreoliwa/nitpick/v0.32.0/mega-linter-plugin-nitpick/nitpick.megalinter-descriptor.yml` to your MegaLinter plugin list but that URL will give you a 404.

Reply:
> but that URL will give you a 404

Yep, I used https://raw.githubusercontent.com/andreoliwa/nitpick/7ef5d58c55f3e9929bcda0e9a8512f9aa848a226/mega-linter-plugin-nitpick/nitpick.megalinter-descriptor.yml to test the PR. 👍🏻 

> Yes, there is a 0.32.0, but there is no MegaLinter plug-in definition on that tag.

Got it. It will be there after I create a new release.